### PR TITLE
new tag added

### PR DESCRIPTION
**Pull request by taufik69**
extra tag added from taufik69

### CODE REVIEW

**Comment by taufik69** [comment]
hey wasim check it out

**Comment by wasimmahamod** [comment]
ok i will check it out

**Comment by taufik69** [comment]
ok bro

**Comment by taufik69** [comment]
i want to add new more feature if you gavv me a permission

Best Regards
Taufik islam

File: script.js
Comment (wasimmahamod) [inline script.js:1]: no need to add taufik add

Comment (taufik69) [reply script.js:1]: ok i will solve it as soon as possibel

**Pull request merged**